lib/profile_api_client.rb

```diff
@@ -5,6 +5,8 @@ class << self
     # TODO: Replace with HTTP requests once the profile API has been built.
 
     def create_school(token:, school:)
+      return { 'id' => school.id, 'schoolCode' => school.code } if ENV['BYPASS_OAUTH'].present?
```

Comment:
Is the BYPASS_OAUTH variable a development thing, so that you don't have to run profile locally?

Reply:
Yep. It's used in a few places to prevent this app communicating with Profile/Hydra/UserInfo API. Although there aren't any checks that we're also in development (or test) environments which would feel a bit safer.